pkg/pipelines/environment_test.go

```diff
@@ -0,0 +1,50 @@
+package pipelines
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestEnv(t *testing.T) {
+
+	gitopsPath, cleanUp := fakeGitopsDir(t)
+	defer cleanUp()
+
+	envParameters := EnvParameters{
+		EnvName:    "dev",
+		GitOpsRepo: "org/gitops",
+		Output:     filepath.Dir(gitopsPath),
+	}
+	if err := Env(&envParameters); err != nil {
+		t.Fatalf("Env() failed :%s", err.Error())
+	}
+
+	wantedPaths := []string{
+		"envs/dev/base/kustomization.yaml",
+		"envs/dev/base/namespace.yaml",
+		"envs/dev/base/rolebinding.yaml",
+		"envs/dev/overlays/kustomization.yaml",
+	}
+
+	for _, path := range wantedPaths {
+		t.Run(fmt.Sprintf("checking path %s", path), func(t *testing.T) {
+			exists, err := isExisting(filepath.Join(gitopsPath, path))
+			assertNoError(t, err)
+			if !exists {
```

Comment:
You could simplify this to something like...

```go
assertFileExists(t, filepath.Join(gitopsPath, path))
```

Reply:
made the required changes